docs/features/pouch_with_lxcfs.md

```diff
@@ -6,6 +6,29 @@ Container solutions provide a general way for users to limit resource usage of r
 
 For example, if we create a general container with limiting 200 MB memory on host of 2 GB memory in total, we can find that resource upper limit is incorrect via command `free`, and it shows all memory size of the host:
 
+Make sure your lxcfs service is running:
```

Comment:
Thanks a lot for your contribution very much. I do think we need this part very much. While I am still wondering if here is the best place to locate the prerequisites of the lxcfs functionablity.

How about adding a third leval title of `Prerequisites` in the second level of `Get Started`? @fanux